### ISSUE TITLE
False positive for silly "private key"

### ISSUE THREAD

**Issue by moui72** [secretlint/secretlint#1235]
**What version of secretlint are you using?**

11.2.4

**What did you do? Please include the actual steps causing the issue.**

```
$ npx @secretlint/quick-start "**/*"
Need to install the following packages:
@secretlint/quick-start@11.2.4
Ok to proceed? (y)


/Users/tyler.peckenpaugh/pager/titan/.github/events/README.md
  115:110  error  [PrivateKey] found private key: ***********************************************************  @secretlint/secretlint-rule-preset-recommend > @secretlint/secretlint-rule-privatekey

✖ 1 problem (1 error, 0 warnings, 0 infos)
```

**What did you expect to happen?**

The error reported is not a real key or even a believable dummy. Here is line 115 from that file:

```shell
   GCP_HARNESS_SA={"type":"service_account","project_id":"your-project","private_key_id":"...","private_key":"-----BEGIN PRIVATE KEY-----\n...\n-----END PRIVATE KEY-----\n","client_email":"your-service@your-project.iam.gserviceaccount.com","client_id":"...","auth_uri":"https://accounts.google.com/o/oauth2/auth","token_uri":"https://oauth2.googleapis.com/token","auth_provider_x509_cert_url":"https://www.googleapis.com/oauth2/v1/certs","client_x509_cert_url":"https://www.googleapis.com/robot/v1/metadata/x509/your-service%40your-project.iam.gserviceaccount.com"}
```

**What actually happened? Please include the actual, raw output from secretlint.**

here's just that file with debugging on, lightly redacted

```sh
$ npx @secretlint/quick-start --debug /Users/tyler.peckenpaugh/pager/titan/.github/events/README.md
  secretlint input: [ '/Users/tyler.peckenpaugh/pager/titan/.github/events/README.md' ] +0ms
  secretlint flags: {
  secretlint   debug: true,
  secretlint   format: 'stylish',
  secretlint   maskSecrets: true,
  secretlint   secretlintignore: '.secretlintignore',
  secretlint   color: true,
  secretlint   terminalLink: true,
  secretlint   cwd: '/Users/tyler.peckenpaugh/pager/titan',
  secretlint   help: false,
  secretlint   version: false,
  secretlint   'no-config': false,
  secretlint   'no-package': false,
  secretlint   '$schema': false,
  secretlint   diff: false,
  secretlint   secretlintrc: '/Users/tyler.peckenpaugh/.npm/_npx/3efd47c6528c435b/node_modules/@secretlint/quick-start/config/.secretlintrc.json'
  secretlint } +1ms
  secretlint cliOptions {
  secretlint   filePathOrGlobList: [ '/Users/tyler.peckenpaugh/pager/titan/.github/events/README.md' ],
  secretlint   outputFilePath: undefined,
  secretlint   ignoreFilePath: '.secretlintignore',
  secretlint   cwd: '/Users/tyler.peckenpaugh/pager/titan'
  secretlint } +0ms
  @secretlint/node Loaded ConfigFilePath: /Users/tyler.peckenpaugh/.npm/_npx/3efd47c6528c435b/node_modules/@secretlint/quick-start/config/.secretlintrc.json +0ms
  @secretlint/node Config: {
  @secretlint/node   sharedOptions: undefined,
  @secretlint/node   rules: [
  @secretlint/node     {
  @secretlint/node       id: '@secretlint/secretlint-rule-preset-recommend',
  @secretlint/node       rule: [Object],
  @secretlint/node       rules: undefined,
  @secretlint/node       options: undefined,
  @secretlint/node       disabled: undefined
  @secretlint/node     }
  @secretlint/node   ]
  @secretlint/node } +0ms
  secretlint search patterns: [ { pattern: '/Users/tyler.peckenpaugh/pager/titan/.github/events/README.md', isDynamic: false } ] +0ms
  secretlint search DEFAULT_IGNORE_PATTERNS: [ '**/.git/**', '**/node_modules/**', '**/.secretlintrc/**', '**/.secretlintrc.{json,yaml,yml,js}/**', '**/.secretlintignore*/**' ] +0ms
  secretlint search ignoreFilePath: .secretlintignore +0ms
  @secretlint/node executeOnFiles file counts: 1 +7ms
  @secretlint/node executeOnFiles file: /Users/tyler.peckenpaugh/pager/titan/.github/events/README.md +0ms
  @secretlint/core source filePath: '/Users/tyler.peckenpaugh/pager/titan/.github/events/README.md' +0ms
  @secretlint/core options: {
  @secretlint/core   configFilePath: '/Users/tyler.peckenpaugh/.npm/_npx/3efd47c6528c435b/node_modules/@secretlint/quick-start/config/.secretlintrc.json',
  @secretlint/core   cwd: '/Users/tyler.peckenpaugh/pager/titan',
  @secretlint/core   formatter: 'stylish',
  @secretlint/core   color: true,
  @secretlint/core   terminalLink: true,
  @secretlint/core   locale: undefined,
  @secretlint/core   maskSecrets: true,
  @secretlint/core   config: { sharedOptions: undefined, rules: [ [Object] ] }
  @secretlint/core } +0ms
  @secretlint/core registerRule @secretlint/secretlint-rule-preset-recommend +0ms
  @secretlint/node executeOnFiles result: { filePath: '/Users/tyler.peckenpaugh/pager/titan/.github/events/README.md', sourceContent: [REDACTED]'GCP_HARNESS_SA={"type":"service_account","project_id":"your-project","private_key_id":"...","private_key":"-----BEGIN PRIVATE KEY-----\\n...\\n-----END PRIVATE KEY-----\\n","client_email":"your-service@your-project.iam.gserviceaccount.com","client_id":"...","auth_uri":"https://accounts.google.com/o/oauth2/auth","token_uri":"https://oauth2.googleapis.com/token","auth_provider_x509_cert_url":"https://www.googleapis.com/oauth2/v1/certs","client_x509_cert_url":"https://www.googleapis.com/robot/v1/metadata/x509/your-service%40your-project.iam.gserviceaccount.com"}\n' + [REDACTED], sourceContentType: 'text', messages: [ { message: 'found private key: ***********************************************************', range: [Array], type: 'message', ruleId: '@secretlint/secretlint-rule-privatekey', ruleParentId: '@secretlint/secretlint-rule-preset-recommend', loc: [Object], severity: 'error', messageId: 'PrivateKey', docsUrl: 'https://github.com/secretlint/secretlint/blob/master/packages/%40secretlint/secretlint-rule-privatekey/README.md#PrivateKey', data: [Object] } ] } +3ms
  @secretlint/node executeOnFiles result counts: 1 +1ms
  @secretlint/formatter formatterName: stylish +0ms
  @secretlint/formatter formatterName: stylish +0ms

/Users/tyler.peckenpaugh/pager/titan/.github/events/README.md
  115:110  error  [PrivateKey] found private key: ***********************************************************  @secretlint/secretlint-rule-preset-recommend > @secretlint/secretlint-rule-privatekey

✖ 1 problem (1 error, 0 warnings, 0 infos)
```
